src/test/java/api/support/fixtures/TemplateContextMatchers.java

```diff
@@ -219,4 +222,16 @@ private static String findItemRepresentationCopyNumber(InventoryItemResource ite
 
     return allOf(loanContextMatcher, itemContextMatcher, loanPolicyMatcher);
   }
+
+  public static Matcher<?> getFeeFineContextMatcher(Account account, FeeFineAction action) {
+    return allOf(
+      hasJsonPath("fee.owner", is(account.getFeeFineOwner())),
+      hasJsonPath("fee.type", is(account.getFeeFineType())),
+      hasJsonPath("fee.actionType", is(action.getTypeAction())),
+      hasJsonPath("fee.actionDateTime", isEquivalentTo(action.getDateAction())),
+      hasJsonPath("fee.actionAmount", closeTo(action.getAmountAction(), 0.001)),
+      hasJsonPath("fee.amount", closeTo(account.getAmount(), 0.001)),
+      hasJsonPath("fee.balance", closeTo(action.getBalance(), 0.001))
+    );
```

Comment:
Why can't we just check that they're equal? Is this because we use doubles instead of BigDecimals?

Reply:
Using just `is()` used to break my tests, but now it's working fine. Replaced.